### PR TITLE
Add available_metrics field in the project type

### PR DESCRIPTION
**Pull request by IvanIvanoff**
#### Summary
```grapqhl
{
  projectBySlug(slug: "sentiment"){ availableMetrics }
}
```

```graphql
{
  "data": {
    "projectBySlug": {
      "availableMetrics": ["devActivity", "ethSpentOverTime", "tokenAgeConsumed", "ethTopTransactions" ...]
    }
  }
}
```

[//]: # (#### Related PRs)

[//]: # (#### Additional deploy notes)

[//]: # (#### Screenshots)

### CODE REVIEW

File: lib/sanbase/model/project/available_metrics.ex [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    end
+  end
+
+  defp social_metrics(%Project{}) do
+    [
+      "socialDominance",
+      "socialGainersLosersStatus",
+      "socialVolume"
+    ]
+  end
```
Comment (valo) [inline lib/sanbase/model/project/available_metrics.ex]: Aren't these metrics available only for a subset of the projects?

Comment (IvanIvanoff) [reply lib/sanbase/model/project/available_metrics.ex]: Yes, this part will  be extended in the next commits - the `socialVolume` is available for `TechIndicators.social_volume_projects()` projects only (will be added). The other stuff - I have to double check

File: lib/sanbase/model/project/available_metrics.ex [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+          }
+        }
+      }
+    }
+    """
+  end
+
+  defp social_volume_projects() do
+    {:ok, projeccts} = Sanbase.TechIndicators.social_volume_projects()
+    projeccts
```
Comment (tspenov) [inline lib/sanbase/model/project/available_metrics.ex]: typo here

Comment (IvanIvanoff) [reply lib/sanbase/model/project/available_metrics.ex]: It has just been fixed with the commit that adds the caching

File: lib/sanbase/model/project/available_metrics.ex [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+  @doc ~s"""
+  Return a list of all GraphQL query names that have an argument `slug`
+  """
+  @spec all :: [String.t()]
+  def all() do
+    project_queries()
+  end
+
+  @doc ~s"""
+  Return a listo of all GraphQL query names that have data for the provided
```
Comment (tspenov) [inline lib/sanbase/model/project/available_metrics.ex]:
```suggestion
  Return a list of all GraphQL query names that have data for the provided
```

Comment (IvanIvanoff) [reply lib/sanbase/model/project/available_metrics.ex]: `listo` sounded good, too 😃

**Pull request merged**